packages/ti_eset/elasticsearch/transform/apt_latest_ioc/transform.yml

```
@@ -0,0 +1,23 @@
+_meta:
+  fleet_transform_version: 0.2.0
+  managed: true
+description: Latest ESET APT IoC data
+dest:
+  index: logs-ti_eset_latest.apt-2
```

```suggestion
  index: logs-ti_eset_latest.apt-1
```
Lets go with `1`? Since this is the first version of the integration?